css-three-column-layout/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="header">
+        <h1>Designed for life science. Optimized for SaMD and Digital Health.</h1>
+      </div>
+    </div>
+    <div class="row">
+
+      <div class="column-one">
+        <div class="white"></div>
```

Comment:
another way to approach this is to nest the image in the div, that way you don't have to use position absolute to position the image over the white background. Flexbox can be used on the `white` background div to control the location of the img.